tools/manifest/requirements.txt

```diff
@@ -0,0 +1 @@
+zstandard==0.17.0
```

Comment:
It's worth noting that this isn't installed for `./wpt manifest-download`. That's a simple change, but the code is already set up to treat this as optional, so I'm not changing that.

Reply:
I think we probably should change that in a followup.

Reply:
Agreed!

Reply:
Done in https://github.com/web-platform-tests/wpt/pull/33452.